packages/react-scripts/config/webpack.config.js

```diff
@@ -440,17 +440,6 @@ module.exports = function (webpackEnv) {
                 ),
                 // @remove-on-eject-end
                 plugins: [
-                  [
-                    require.resolve('babel-plugin-named-asset-import'),
```

Comment:
Is this plugin not used anymore?

Reply:
I'll have to look into it - don't remember the details anymore

Reply:
We could look into updating that package to webpack 5, but not sure what the package does and if still needed?

Reply:
(looking at the tests for `babel-plugin-named-asset-import` it seems focused on svg import etc. but afaik it should work as expected in the wp5, we do have a couple of e2e tests for that as I remember)

Reply:
@iansu I'll resolve this for now, it could be picked up in #11213 *(if we face issues with svg imports)*